lib/active_pivot/api/project.rb

```diff
@@ -2,7 +2,7 @@ module ActivePivot
   module Api
     class Project < OpenStruct
       def self.all
-        Api::PaginatedCollection.new("/services/v5/projects.json").all.map do |remote_project|
+        Api::PaginatedCollection.new("/services/v5/projects.json?fields=:default,current_velocity,current_volatility").all.map do |remote_project|
           self.new(remote_project) rescue nil
```

Comment:
:put_litter_in_its_place: 

`rescue nil` is dangerous stuff, I know it was pulled from relapse but would be nice if we had a smarter way to handle errors in there